### PR TITLE
Fix decal register + obsolete cookie API

### PR DESCRIPTION
**Pull request by sebastienlagarde**
### Purpose of this PR
This PR fix decal when not using the option Metal and ambient occlusion.
I was introduce by the DXC support PR that is forcing to have explicit register.

It also fix obsolete API use for cookie baking.

Added a test for 3RT decal (with  "Metal and ambient occlusion" disabled)

+
Fix tooltip for color adjustement

QA: Please check that Decal for projector, Mesh and emissive with both case

---
### Testing status

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in developer mode, you have a button at end of resources that check the paths)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/Material/Decal/ClearPropertyMaskBuffer.compute [hunk 1]
```diff
@@ -5,14 +5,20 @@
 
 #pragma kernel CSMain
 
+#ifdef PLATFORM_SUPPORTS_BUFFER_ATOMICS_IN_PIXEL_SHADER
```
Comment (CifaCia) [inline com.unity.render-pipelines.high-definition/Runtime/Material/Decal/ClearPropertyMaskBuffer.compute:8]: Question: do we support any platform that doesn't have this defined? This is defined for D3D11/12, Vulkan and I assume on Console it is too.

Comment (FrancescoC-unity) [inline com.unity.render-pipelines.high-definition/Runtime/Material/Decal/ClearPropertyMaskBuffer.compute:8]: Question: do we support any platform that doesn't have this defined? This is defined for D3D11/12, Vulkan and I assume on Console it is too.

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Runtime/Material/Decal/ClearPropertyMaskBuffer.compute:8]: wasn't on Metal at some point, so for now I keep it :)

Comment (FrancescoC-unity) [reply com.unity.render-pipelines.high-definition/Runtime/Material/Decal/ClearPropertyMaskBuffer.compute:8]: It is defined on it now, but sure doesn't hurt.

**Comment by sebastienlagarde** [comment]
Yamato is still not green but error come from HDRP/staging, merging.

**Comment by sebastienlagarde** [comment]
QA have validated on PS4. Still waiting for QA for decal coverage but merged as it is needed to unlock HDRP/staging which need cookie lightmap fix to be able to compile

**Pull request merged**